.circleci/config.yml

```diff
@@ -137,3 +137,33 @@ jobs:
       # Store test results from previous step.
       - store_test_results:
           path: /tmp/test-results
+
+workflows:
+  version: 2
+  build:
+    jobs:
+    - build
+
+  nightly:
+    triggers:
+    - schedule:
+        cron: "0 0 * * *"
+        filters:
+          branches:
+            only:
+            - master
+    jobs:
+    - build:
+        context: nightly
+
+  twitter_develop:
+    triggers:
+    - schedule:
+        cron: "0 0 * * *"
+        filters:
+          branches:
+            only:
+            - master
```

Comment:
Recommend setting `twitter-develop` to `10 12 * * *` (5:10am PT) and `nightly` to `30 12 * * *` (5:30am PT). The intent is that when `twitter-develop` is broken, CI is only red for about 20 minutes in the early morning before `master` makes it go green.

Reply:
TIL! Thanks @siggy, got it updated now.